services/graph/apps/api/src/metadata.ts

```diff
@@ -1,107 +1,15 @@
-            },
-          },
-        ],
-        [
-          import('../../../libs/common/src/dtos/user-graph.dto'),
-          {
-            UserGraphDto: {
-              dsnpId: { required: true, type: () => String },
-              dsnpGraphEdges: { required: false, type: () => [t['../../../libs/common/src/dtos/dsnp-graph-edge.dto'].DsnpGraphEdge] },
-            },
-          },
-        ],
-        [import('../../../libs/common/src/dtos/graph-change-response.dto'), { GraphChangeRepsonseDto: { referenceId: { required: true, type: () => String } } }],
-        [
-          import('../../../libs/common/src/dtos/graph-query-params.dto'),
-          {
-            GraphsQueryParamsDto: {
-              dsnpIds: { required: true, type: () => [String] },
-              privacyType: { required: true, enum: t['../../../libs/common/src/dtos/privacy-type.enum'].PrivacyType },
-              graphKeyPairs: { required: false, type: () => [t['../../../libs/common/src/dtos/graph-key-pair.dto'].GraphKeyPairDto] },
-            },
-          },
-        ],
-        [
-          import('../../../libs/common/src/dtos/provider-graph.dto'),
-          {
-            ProviderGraphDto: {
-              dsnpId: { required: true, type: () => String },
-              connections: { required: true, type: () => ({ data: { required: true, type: () => [t['../../../libs/common/src/dtos/connection.dto'].ConnectionDto] } }) },
-              graphKeyPairs: { required: false, type: () => [t['../../../libs/common/src/dtos/graph-key-pair.dto'].GraphKeyPairDto] },
-              webhookUrl: { required: false, type: () => String },
-            },
-          },
-        ],
-        [
-          import('../../../libs/common/src/dtos/watch-graphs.dto'),
-          { WatchGraphsDto: { dsnpIds: { required: false, type: () => [String] }, webhookEndpoint: { required: true, type: () => String } } },
-        ],
-      ],
-      controllers: [
-        [import('./controllers/health.controller'), { HealthController: { healthz: {}, livez: {}, readyz: {} } }],
-        [
-          import('./controllers/v1/webhooks-v1.controller'),
-          {
-            WebhooksControllerV1: {
-              getAllWebhooks: { type: Object },
-              getWebhooksForMsa: { type: [String] },
-              getWebhooksForUrl: { type: [String] },
-              watchGraphs: {},
-              deleteAllWebhooks: {},
-              deleteWebhooksForMsa: {},
-              deleteAllWebhooksForUrl: {},
-            },
-          },
-        ],
-        [
-          import('./controllers/v1/graph-v1.controller'),
-          {
-            GraphControllerV1: {
-              getGraphs: { type: [t['../../../libs/common/src/dtos/user-graph.dto'].UserGraphDto] },
-              updateGraph: { type: t['../../../libs/common/src/dtos/graph-change-response.dto'].GraphChangeRepsonseDto },
-            },
-          },
-        ],
-      ],
-    },
-  };
-};
+    const t = {
```

Comment:
nit: npm run format will also fix up this file

Reply:
I'll format this one file, but I want to do a separate PR to just `npm run format` all of the services using the common prettier config, so it doesn't pollute a PR with actual logic changes in it.